### PR TITLE
BETA: Register xchillds.is-a.dev

### PR DESCRIPTION
**Pull request by fugoou**
Added `xchillds.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/xchillds.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "XChillDS",
+        "email": "tsukasachan666@gmail.com"
+    },
+    "record": {
+        "CNAME": "xchillds.github.io"
```
Comment (DEV-DIBSTER) [inline domains/xchillds.json:7]: Redirect loop.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by DEV-DIBSTER** [comment]
@XChillDS The CNAME still redirects loop, fix this to get this merged.

**Comment by XChillDS** [comment]
> @XChillDS The CNAME still redirects loop, fix this to get this merged.

what redirect loop, I don't understand 🥲

**Comment by DEV-DIBSTER** [comment]
> > @XChillDS The CNAME still redirects loop, fix this to get this merged.
> 
> what redirect loop, I don't understand 🥲

Go the the value you gave of xchillds.github.io. It then redirects to xchillds.is-a.dev, which is what you're trying to register. xchillds.is-a.dev with the CNAME you provide would go back to the GitHub. Endless loop cycle. Nothing is being displayed.

**Comment by XChillDS** [comment]
saya sudah melakukan nya

**Comment by XChillDS** [comment]
> > > @XChillDS The CNAME still redirects loop, fix this to get this merged.
> > 
> > 
> > what redirect loop, I don't understand 🥲
> 
> Go the the value you gave of xchillds.github.io. It then redirects to xchillds.is-a.dev, which is what you're trying to register. xchillds.is-a.dev with the CNAME you provide would go back to the GitHub. Endless loop cycle. Nothing is being displayed.

I Will do it

**Pull request closed without merging**